micro-couchbase/src/main/java/com/aol/micro/server/couchbase/CouchbaseDistributedMapClient.java

```diff
@@ -3,10 +3,10 @@
 import java.util.Optional;
 import java.util.concurrent.ExecutionException;
 
+import com.aol.cyclops2.util.ExceptionSoftener;
```

Comment:
It's non-core, cyclops is very big. The core data types are defined in cyclops.X, everything else stays as com.aol.cyclops, and internal stuff is at com.aol.cyclops.internal.